include/deal.II/multigrid/mg_transfer_global_coarsening.templates.h

```diff
@@ -893,6 +934,10 @@ namespace internal
                   cell->get_mg_dof_indices(indices);
 
                 buffer.push_back(cell->active_fe_index());
+                if (cell->level() != 0)
+                  buffer.push_back(cell->parent()->refinement_case());
+                else
+                  buffer.push_back(numbers::invalid_unsigned_int);
```

Comment:
Is `invalid_unsigned_int` the right number here? I'm asking because `buffer` storing the argument passed in is of type `types::global_dof_index` and `refinement_case` is of type `std::uint8_t`. I would pick either `types::invalid_dof_index` or `static_cast<std::unit8_t>(-1)`.

Reply:
Changed to `numbers::invalid_dof_index`.